safehttp/safehttptest/recorder.go

```diff
@@ -26,6 +28,17 @@ import (
 
 type testDispatcher struct{}
```

Comment:
Does this mean you could use the `DefaultDispatcher` here now? Possibly in other tests too?

This can be done in a follow-up PR.

Reply:
Yep, I think so, and it would definitely make tests cleaner. We could use the `DefaultDispatcher` in tests by default (too many defaults in one sentence) and only create a `testDispatcher` for analysing some custom behaviour. 

But yeah, should be done in a clean-up PR and not in this one, I think.

Reply:
Opened #149